assets/prefabs/game/gameEntity.prefab

```diff
@@ -0,0 +1,5 @@
+{
+    "LASConfig" : {
```

Comment:
This component is defined in LigthAndShadow, i.e., this introduces a dependency **LightAndShadowResources ⟶ LightAndShadow**, but actually the dependency order is the other way around.

I would say the game state entity prefab is tightly coupled to the game mode, so I would suggest to just move this prefab over to LightAndShadow and include it in https://github.com/Terasology/LightAndShadow/pull/203